Documentation/proposals/202411-zone-aware-sharding.md

```diff
@@ -0,0 +1,295 @@
+# Zone aware sharding for prometheus
```

Comment:
@simonpasquier should we highlight about additionalScrapeConfigs not working out of the box with this solution?

Reply:
yes I'd probably put in in the non-goals section.

Reply:
Can you add a suggestion on what to put there?

Reply:
added